api/devicerepository.proto

```diff
@@ -0,0 +1,366 @@
+    string part_number = 3;
+  }
+  // Available hardware versions.
+  repeated HardwareVersion hardware_versions = 5;
+
+  message FirmwareVersion {
+    message Profile {
+      // Profile identifier, as defined in the Device Repository.
+      string profile_id = 1 [(gogoproto.customname) = "ProfileID"];
+      // Whether the device is LoRaWAN certified.
+      bool lorawan_certified = 2 [(gogoproto.customname) = "LoRaWANCertified"];
+      // Payload formatter codec identifier, as defined in the Device Repository.
+      string codec_id = 3 [(gogoproto.customname) = "CodecID"];
+    }
+    // Firmware version string.
+    string version = 1;
+    // Numeric firmware revision number.
+    uint32 numeric = 2;
+    // Hardware versions supported by this firmware version.
+    repeated string supported_hardware_versions = 3;
+    // Device profiles for each supported region (band).
+    map<string, Profile> profiles = 4;
+  }
+  // Available firmware versions.
+  repeated FirmwareVersion firmware_versions = 6;
+  // List of sensors included in the device.
+  repeated string sensors = 7;
+  message Dimensions {
+    // Device width (mm).
+    google.protobuf.FloatValue width = 1;
+    // Device height (mm).
+    google.protobuf.FloatValue height = 2;
+    // Device diameter (mm).
+    google.protobuf.FloatValue diameter = 3;
+    // Device length (mm).
+    google.protobuf.FloatValue length = 4;
+  }
+  // Device dimensions.
+  Dimensions dimensions = 8;
+  // Device weight (gram).
+  google.protobuf.FloatValue weight = 9;
+  message Battery {
+    // Whether the device battery can be replaced.
+    google.protobuf.BoolValue replaceable = 1;
+    // Battery type.
+    string type = 2;
+  }
+  // Device battery information.
+  Battery battery = 10;
+  message OperatingConditions {
+    message Limits {
+      // Min value of operating conditions range.
+      google.protobuf.FloatValue min = 1;
+      // Max value of operating conditions range.
+      google.protobuf.FloatValue max = 2;
+    }
+    // Temperature operating conditions (Celsius).
+    Limits temperature = 1;
+    // Relative humidity operating conditions (Fraction, in range [0, 1]).
+    Limits relative_humidity = 2;
+  }
+  // Device operating conditions.
+  OperatingConditions operating_conditions = 11;
+  // Device IP rating code.
+  string ip_code = 12 [(gogoproto.customname) = "IPCode"];
+  // Supported key provisioning methods.
+  repeated KeyProvisioning key_provisioning = 13;
+  // Device key security.
+  KeySecurity key_security = 14;
+
+  message Photos {
+    // Main device photo.
+    string main = 1;
+    // List of other device photos.
+    repeated string other = 2;
+  }
+  // Device photos.
+  Photos photos = 15;
+  message Videos {
+    // Link to main device video.
+    string main = 1;
+    // Links to other device videos.
+    repeated string other = 2;
+  }
+  // Device video.
+  string video = 16;
+  // Device videos.
+  Videos videos = 17;
```

Comment:
I still don't get `video` field 16

Reply:
It seems to be part of the example windsensor device, but indeed I do not see the `video` field to be a part of the schema itself. 

https://github.com/TheThingsNetwork/lorawan-devices/blob/master/vendor/example/windsensor.yaml#L119

Will remove, but we need to fix the example as well.

Reply:
https://github.com/TheThingsNetwork/lorawan-devices/pull/53